### DISCUSSION TITLE
More Workshop Tags

### DISCUSSION THREAD

**Discussion by Crazyjr20** [FakeFishGames/Barotrauma#14932] [Ideas and suggestions]
The current tags on the workshop are very limiting and barely help to sort mods.
I would assume adding new tags isn't super difficult so I'd like to suggest we do that.

First off, one of, if not the biggest modding section in this game is Submarines, but we only have one tag for that, so I suggest we split _"Submarine"_ into: 
>_Submarine
Outpost 
Beacons
Wrecks
 Ruin._

Though perhaps _Wrecks/Outposts/Ruins_ could be one tag to avoid bloat as they all fill a similar spot.

I'd say the second one to tackle is Items, but this is a bit harder due to the varied nature of its contents.
Perhaps splitting it into:
>_Item
Weapons
Equipment
Medical_ 

I believe keeping the broader _Item_ tag for things that dont fit any category would be a good idea while letting people refine the results with the other tags.

I also think we should get a _PvP_ tag, so people can search for _"submarine, pvp"_ for pvp focused submarines, or _"outpost, pvp"_ for outpost deathmatch maps.

Next, Mods that require Lua or C for Barotrauma should probably have their own tag, but I understand that you are working on your own implementation for these kind of mods unless I'm mistaken, so might be better to wait until then.

**Comment by SomeRandomNoobKekeke** [comment]
Also tags suggested in [this discussion](https://discord.com/channels/103209684680323072/1280525906007293985):
Gameplay Mechanics, QOL, Server side, Client side, Outdated

And ofc you need to hire mr. Crazyjr20 to check 60k workshop items and assign correct tags :BaroDev:

**Reply by Crazyjr20** [reply]
Modders try not to select every tag challenge: IMPOSSIBLE

**Comment by Ydrec** [comment]
Considering we are eventually getting new modding system, this should be relevant even more, 
Currently we have

```
-  Submarine
-  Mission
-  Item
-  Item Assembly
-  Monster
-  Language
-  Event set
-  Total conversion
-  Art
-  Environment
```

Some crucial additional tags would be
```
-  Code/Plugin (aka uses new modding system or luacs)
-  Clientside
-  Serverside
-  PVP (mods meant for pvp)
-  Gamemode
```

Some extra tags for content type would also help like
```
-  Character
-  Job/s
-  QOL
-  Gameplay
-  Combat
```
